gdsfactory/get_netlist.py

```diff
@@ -231,7 +232,10 @@ def get_netlist(
             continue
 
         c = reference.cell
-        origin = reference.dtrans.disp
+        if hasattr(reference, "dtrans"):
+            origin = reference.dtrans.disp
+        else:
+            origin = reference.trans.disp
         x = origin.x
```

Comment:
Just go for reference.dcplx_trans here. I have updated kfactory which should always allow that. The PR is still open as some stuff changed in the routing placer which I don't fully understand yet